kotlin-service/src/test/kotlin/com/microservice/kotlin/KotlinAppIntegrationTest.kt

```diff
@@ -0,0 +1,41 @@
+package com.microservice.kotlin
+
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.MediaType
+import org.springframework.security.test.context.support.WithMockUser
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.*
+
+@ExtendWith(SpringExtension::class)
+@SpringBootTest(classes = [KotlinApp::class], properties = ["configuration.swagger=false", "debug=true", "logging.level.org.springframework.security=debug"])
+@AutoConfigureMockMvc
+class KotlinAppIntegrationTest(@Autowired val client: MockMvc) {
+
+    @Test
+    @DisplayName("Test - When Calling GET - /api/tasks should return list of tasks and response 200 - OK")
+    @WithMockUser(roles = ["TASK_READ"], username = "rodrigo")
+    fun shouldReturnListOfTasksWhenCallingApi() {
```

Comment:
these test function names can be changed to:
```
fun `Should return list of Tasks When calling Api`() {
```

please see in [code conventions](https://kotlinlang.org/docs/reference/coding-conventions.html):
> In tests (and only in tests), it's acceptable to use method names with spaces enclosed in backticks.